lib/internal/event_target.js

```diff
@@ -78,6 +78,13 @@ const isTrusted = ObjectGetOwnPropertyDescriptor({
   }
 }, 'isTrusted').get;
 
+const descriptor = {
```

Comment:
```suggestion
const trustedDescriptor = {
```

or something similar that is more descriptive.